airtable/assets/index.html

```diff
@@ -42,21 +42,24 @@ <h1>Welcome to your new Twilio App</h1>
         <h3>Get started with your app</h3>
         <p>This app automatically adds people who text in to your phone number into the specified table in Airtable. Follow the steps below to test your app:</p>
         <ol>
-          <li>Text your Twilio phone number (referenced below) with a message.</li>
+          <li>Text your Twilio phone number with a message.</li>
           <li>Go to the specified table in Airtable and refresh it. Your message and phone number should now be input into the table.</li>
-          <li>In order to test the broadcast feature, go to /broadcast-sms. All numbers that are in Airtable should receive a text.</li>
+          <li>In order to test the broadcast feature, go to   <p>
+              <pre><code><span class="function-root"></span>/broadcast-sms</code></pre>
```

Comment:
In some of the other pages i've done something like `<li><a href="/broadcast-sms">Click here to test the broadcast feature</a></li>`